mistlib/core.py

```diff
@@ -399,6 +399,80 @@ def append_file(input_filename, output_filename):
                 output_file.write(input_content)
     
             return
+    
+    def write_additivefoam_input(self, file):
+        specific_heat_1 = None
+        specific_heat_2 = None
+        # Reference temperature based on a property from self.properties
+        comment_block = """/*---------------------------------------------------------------------------
```

Comment:
Multi-line string formatting issue? We could add in black or yapf to auto-format

Reply:
Opened #19